### PR TITLE
Add starccm+

### PR DESCRIPTION
**Pull request by JonShelley**
Added files and instructions to run starccm+

### CODE REVIEW

File: apps/starccm/readme.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+## Install and run Fluent Benchmarks
```
Comment (kanchanm) [inline apps/starccm/readme.md]: StarCCM

File: apps/starccm/readme.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+## Prerequisites
+
+Cluster is built with the desired configuration for networking, storage, compute etc. 
+
+Recommended cluster setup
+Start with the simple_hpc_pbs example. Before you do azhpc-build copy the scripts directory from <azurehpc>/apps/starccm to your cluster build directory. First, you will need to add the following line above the "tags" section for the headnode.
+
+"data_disks": [2048, 2048],
+    
+ Second, add the following section in the scripts section above the pbsdownload piece (~line 90 in the config.json file).
```
Comment (kanchanm) [inline apps/starccm/readme.md]: an alternative (preferred) to requiring to do this during cluster setup would be to run the script on the compute nodes post-setup. see https://github.com/Azure/azurehpc/tree/master/apps/lammps for e.g.

File: apps/starccm/run_civil_ompi4.pbs [hunk 3]
```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+#PBS -N civil
+#PBS -l select=2:ncpus=60:mpiprocs=60:mem=220gb
+#PBS -l walltime=168:0:0
+#PBS -k oed
+
+MPI=openmpi-4.0.0
+VERSION=14.06.004
+AZ_INST=HB60rs
```
Comment (kanchanm) [inline apps/starccm/run_civil_ompi4.pbs]: not used..

File: apps/starccm/readme.md (hunk 1 above)
Comment (JonShelley) [reply apps/starccm/readme.md]: Fixed

File: apps/starccm/readme.md (hunk 2 above)
Comment (JonShelley) [reply apps/starccm/readme.md]: Addressed as recommended by the alternative approach

**Pull request merged**